packages/documentation/gatsby-config.js

```diff
@@ -86,5 +66,51 @@ module.exports = {
     `gatsby-transformer-remark`,
     `github-api-pages`,
     `mdx-pages`,
+    {
+      resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
+      options: {
+        // Fields to index
+        fields: [`title`, `tags`],
+        // How to resolve each field`s value for a supported node type
+        resolvers: {
+          // For any node of type MarkdownRemark, list how to resolve the fields` values
+          SitePage: {
+            title: node => {
+              if (node.context && node.context.frontmatter && node.context.frontmatter.title) {
```

Comment:
lodash get or optional chaining would make this cleaner- not sure how to fix this upstream when the nodes are created